src/crypto/public_key/elliptic/ecdsa.js

```diff
@@ -25,25 +25,52 @@
 
 'use strict';
 
+import BN from 'bn.js';
+import ASN1 from 'asn1.js';
+import jwkToPem from 'jwk-to-pem';
+
 import curves from './curves.js';
 import BigInteger from '../jsbn.js';
+import config from '../../../config';
+import enums from '../../../enums.js';
+import util from '../../../util.js';
+import base64 from '../../../encoding/base64.js';
+
+const webCrypto = util.getWebCrypto();
+const webCurves = curves.webCurves;
+const nodeCrypto = util.getNodeCrypto();
+const nodeCurves = curves.nodeCurves;
+
+var ECDSASignature = ASN1.define('ECDSASignature', function() {
+  this.seq().obj(
+    this.key('r').int(),  // FIXME int or BN?
```

Comment:
Can you explain the FIXME here?

Reply:
At the time I needed to confirm that the `asn1.js` package handes big integers correctly, which it does, so we can remove this.